runtime/vm/vmifunc.c

```diff
@@ -192,7 +192,7 @@ vmi_getPortLibrary(VMInterface* vmi)
  * <TABLE>
  * <TR><TD><B>Property Name</B></TD>			<TD><B>Example Value or Description</B></TD></TR>
  * <TR><TD>java.vendor</TD>			<TD>"Eclipse OpenJ9"</TD></TR>
- * <TR><TD>java.vendor.url</TD>			<TD>"https://www.eclipse.org/openj9"</TD></TR>
+ * <TR><TD>java.vendor.url</TD>			<TD>"https://eclipse.dev/openj9"</TD></TR>
```

Comment:
The URL should also be updated in the extensions repositories; for example, these definitions found in jdk11:
```
VENDOR_URL="http://www.eclipse.org/openj9"
OPENJ9_BASE_URL := https://www.eclipse.org/openj9/
```

Reply:
Working on it. Newer repos don't define VENDOR_URL, but the system property is still set to an openj9 URL, I'm not sure where it comes from.